verify/src/main/java/com/twilio/verify/domain/factor/FactorFacade.kt

```diff
@@ -147,6 +147,17 @@ internal class FactorFacade(
     }
   }
 
+  fun clearLocalData(success: () -> Unit) {
+    execute(success, {}) { onSuccess, _ ->
+      try {
+        pushFactory.deleteAllFactors(onSuccess)
+      } catch (e: Exception) {
+        factorProvider.clearLocalData()
```

Comment:
We have the key for encrypted storage in the same keyStorage (AndroidKeyStore), so we cannot delete all the key pairs. Also, we will have other key pairs, for example for authentication JWTs for other factor types. The instance was created, so the encrypted storage already exists, if I want to add a new factor it will fail because the key for encryption doesn't exist

Reply:
oh I see, but in case the catch block is executed, the Key pairs would remain in the AndroidKeyStore. Is there any concern about this apart from the little disk space it would take?

Reply:
Yes, that's the only concern, because the key pair alias is a random value, so there is not an easy way to know the corresponding factor.

Reply:
Ok, thanks!